server/channels/jobs/batch_worker_test.go

```diff
@@ -21,7 +21,7 @@ func TestBatchWorkerRace(t *testing.T) {
 
 	th := Setup(t)
 
-	worker := jobs.MakeBatchWorker(th.Server.Jobs, th.Server.Store(), 1*time.Second, func(rctx *request.Context, job *model.Job) bool {
+	worker := jobs.MakeBatchWorker(th.Server.Jobs, th.Server.Store(), 1*time.Second, func(rctx request.CTX, job *model.Job) bool {
```

Comment:
I'm assuming you've made sure that changing the types isn't accidentally breaking something? :)